Dockerfile.spark

```diff
@@ -36,6 +36,10 @@ ENV LIVY_BUILD_PATH /apps/build/livy
 RUN cd /apps/build && \
 	git clone https://github.com/apache/incubator-livy.git livy && \
 	cd $LIVY_BUILD_PATH && \
+  curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && \
```

Comment:
Move this pip installation and setup tools upgrade to line 15, so we don't duplicate these steps.  

```Dockerfile
# Update this
RUN easy_install3 pip py4j
RUN pip install --upgrade setuptools
```

Reply:
I cannot get the build to work locally to verify, so please re-build and verify your fix after making this change!

Reply:
I am on it

Reply:
I moved the changes to the RUN command on line 15 and have built it without any issues. One thing I would like to mention/for future reference is I am adding this separate setuptools upgrade for Livy build only. PIP installed by easy_install3 is on Python 3.5 and Livy uses the default 2.7 to build the API.

Logs:
```
Successfully built aa4936a99098
Successfully tagged jupyter/sparkmagic-livy:latest
```